src/transforms/posthtml.js

```diff
@@ -11,20 +21,28 @@ module.exports = async function(asset) {
   let res = await posthtml(config.plugins).process(asset.ast, config);
 
   asset.ast = res.tree;
-  asset.isAstDirty = true;
-};
+  asset.isAstDirty = true
+}
 
 async function getConfig(asset) {
-  let config = await asset.getConfig(
-    ['.posthtmlrc', '.posthtmlrc.js', 'posthtml.config.js'],
-    {packageKey: 'posthtml'}
-  );
+  let config =
+    asset.getPackage().posthtml ||
+    (await asset.getConfig([
+      '.posthtmlrc',
+      '.posthtmlrc.js',
+      'posthtml.config.js'
+    ]));
   if (!config && !asset.options.minify) {
     return;
   }
 
   config = Object.assign({}, config);
   config.plugins = await loadPlugins(config.plugins, asset.name);
   config.skipParse = true;
-  return config;
+  return config
```

Comment:
Looks like Prettier did not format (`--no-verify`?), you can manually run it by running `yarn format` or `npm run format`.

Reply:
Did not know this before, updated